koch.nim

```diff
@@ -352,8 +354,8 @@ proc winReleaseArch(arch: string) =
     # Rebuilding koch is necessary because it uses its pointer size to
     # determine which mingw link to put in the NSIS installer.
     nimexec "c --cpu:$# koch" % cpu
-    exec "koch boot -d:release --cpu:$#" % cpu
-    exec "koch --latest zip -d:release"
+    exec "./koch boot -d:release --cpu:$#" % cpu
```

Comment:
Every ``exec "./koch ..."`` should be turned into ``cook "..."``.

Reply:
fixed with `kochExec` which is more consistent with nimexec. not everyone speaks german ;-)